operator_ui/@types/core/store/models.d.ts

```diff
@@ -51,6 +51,7 @@ declare module 'core/store/models' {
     tasks: TaskSpec[]
     startAt: nullable.Time
     endAt: nullable.Time
+    name: string
```

Comment:
Should this be `name?`

Reply:
We get an empty string if the name is undefined